web/src/main/java/se/inera/intyg/webcert/web/service/facade/impl/certificatefunctions/CertificateSignAndSendDescriptionFunctionImpl.java

```diff
@@ -0,0 +1,119 @@
+package se.inera.intyg.webcert.web.service.facade.impl.certificatefunctions;
+
+import static se.inera.intyg.common.support.facade.model.CertificateRelationType.COMPLEMENTED;
+
+import java.util.Optional;
+import org.springframework.stereotype.Component;
+import se.inera.intyg.common.af00213.support.Af00213EntryPoint;
+import se.inera.intyg.common.db.support.DbModuleEntryPoint;
+import se.inera.intyg.common.doi.support.DoiModuleEntryPoint;
+import se.inera.intyg.common.fk7263.support.Fk7263EntryPoint;
+import se.inera.intyg.common.lisjp.support.LisjpEntryPoint;
+import se.inera.intyg.common.luae_fs.support.LuaefsEntryPoint;
+import se.inera.intyg.common.luae_na.support.LuaenaEntryPoint;
+import se.inera.intyg.common.luse.support.LuseEntryPoint;
+import se.inera.intyg.common.support.facade.model.Certificate;
+import se.inera.intyg.common.ts_bas.support.TsBasEntryPoint;
+import se.inera.intyg.common.ts_diabetes.support.TsDiabetesEntryPoint;
+import se.inera.intyg.infra.security.authorities.AuthoritiesHelper;
+import se.inera.intyg.infra.security.common.model.AuthoritiesConstants;
+import se.inera.intyg.webcert.web.web.controller.facade.dto.ResourceLinkDTO;
+import se.inera.intyg.webcert.web.web.controller.facade.dto.ResourceLinkTypeDTO;
+
+@Component
+public class CertificateSignAndSendDescriptionFunctionImpl implements CertificateSignAndSendDescriptionFunction {
```

Comment:
I think you can shorten the name by removing "Description". Even though the bug was that the description was wrong, the Function is the whole SignAndSend-functionality and not just the description.